stdpopsim/generic_models.py

```diff
@@ -0,0 +1,87 @@
+# generic_models
+import msprime
+import stdpopsim.models as models
+
+
+#########
+# Generic Demographic Models
+#
+#
+########
+
+class generic_constant_size(models.Model):
+    """
+    Model Name:
+        generic_constant_size
+
+    Model Description:
+        single-size, single-population model
+
+    Model population indexes:
+        - Population1 : 0
+
+    Parameter Table:
+        .. csv-table::
+            :widths: 15 8 20
+            :header: "Parameter Type (units)", "Value", "Description"
+            :file: ../docs/parameter_tables/homo_sapiens/GutenkunstThreePopOutOfAfrica_params.csv
+
+    CLI help:
+        python -m stdpopsim homo-sapiens generic_constant_size -h
+
+    Citation:
+
+    """  # noqa: E501
+
+    def __init__(self, N=100):
```

Comment:
Having a default seems pretty arbitrary here. I'd say we force people to choose an N.

Reply:
so I needed to set defaults to deal with the tests that use `models.all_models()`. `all_models()` works fine for all of the organism specific models because they carry around their own parameters. For the generics, I went with defaults to get around that. suggestions?

Reply:
Hmm. Maybe we should make the default 1 then so, so that we're working in coalescent units by default like msprime?

Reply:
easy enough